app/src/main/res/values/strings.xml

```diff
@@ -270,7 +270,7 @@
     <string name="sort_by_times_opened">Times opened</string>
     <string name="sort_by_mobile_data">Mobile data</string>
     <string name="pref_import_export_blocking_rules">Import/export blocking rules</string>
-    <string name="pref_import_export_blocking_rules_msg">Import/export rules, import external rules from Watt or Blocker.</string>
+    <string name="pref_import_export_blocking_rules_msg">You can import rules exported from Watt or Blocker.</string>
```

Comment:
Is this a summary of what this option reveals? This is a very misleading summary. Revert it back if you're running out of ideas.